URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -0,0 +1,421 @@
+//
+//  URBNSwAlertStyler.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/22/17.
+//
+//
+
+import Foundation
+import URBNConvenience
+
+public struct URBNSwAlertStyler {
+    
+    /**
+     * Pass no to disable blurring in the background
+     */
+    public var blurEnabled = true
```

Comment:
When doing `TextStyle` for whitelabel, I found it useful to group related things in a type. e.g. in the this case you could create:
```swift
struct Blur {
    let color: UIColor
    let radius: CGFloat
}
```

and then use an optional to figure out if there is a blur:
```swift
    public var blur: Blur? = Blur(color: UIColor.white.withAlphaComponent(0.4), radius: 5)
```